core/unit_test/cuda/TestCuda_DebugPinUVMSpace.cpp

```diff
@@ -34,8 +34,8 @@ struct CopyFunctor {
 };
 
 TEST(cuda, debug_pin_um_to_host) {
-#ifdef KOKKOS_ARCH_AMPERE87
-  GTEST_SKIP() << "skipping for Jetson devices that have integrated memory";
+#if defined(KOKKOS_ARCH_AMPERE87) || defined(KOKKOS_ARCH_HOPPER90)
+  GTEST_SKIP() << "skipping for devices that have integrated memory";
```

Comment:
I don't understand that change.
Are we trying to also skip for GH200 (in which case the macro guard does not look right to me)?
I am not familiar with the "integrated memory" terminology. Can you share a reference?